modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -160,6 +162,8 @@ GLOBAL_LIST_EMPTY(ghost_records)
 
 /obj/machinery/cryopod/Initialize(mapload)
 	..()
+	if(!quiet && src.z != 2) // maybe I should change the list to not_quiet_cryopods and dont block interlink cryo?
```

Comment:
I think your comment is valid, and you could just make the list be `GLOB.valid_cryopods` instead.
```suggestion
	if(!quiet)
```